site/content/news/rdf4j-400.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "RDF4J 4.0.0 released"
+date: 2022-04-22T21:07:16+0200
+layout: "single"
+categories: ["news"]
+---
+
```

Comment:
Need a hand filling this out?

Reply:
As always I appreciate your help with this :)

Reply:
Ok I'll write a blurb and push it.

Reply:
Done. I added some balloons (because, you know, we can).